Gopkg.toml

```diff
@@ -17,3 +17,7 @@
 [[constraint]]
   name = "google.golang.org/grpc"
   version = "1.4.3"
+
+[[constraint]]
+  name = "github.com/lightstep/thrift"
+  branch = "0.9.2"
```

Comment:
How about pointing resetting `master` of `github.com/lightstep/thrift` to our forked 0.9.2?

I've been planning to add `go.mod` files, but wanted to do that separately since I foresee some near-term churn with the packages. The main package has a lot of internal and/or test-only code right now, so I'm experimenting with some different ways to trim it down.

Were you or anyone you're aware of planning to use the tracer with go modules in the super near future?